### PR TITLE
Changes after review

### PR DESCRIPTION
**Pull request by JJonka**

### CODE REVIEW

File: src/components/Box/Box.test.tsx [hunk 1]
```diff
@@ -4,11 +4,18 @@ import "@testing-library/jest-dom";
 import Box from "./Box";
 
 describe("Box", () => {
-  it("text 'Box' should be rendered", () => {
+  it("child should be rendered properly", () => {
```
Comment (FutureInventor) [inline src/components/Box/Box.test.tsx:7]: "it" basically meant "this test", therefore "it('should render the children properly')" is the proper description. Change other descriptions as well.

File: src/components/Card/Card.tsx [hunk 2]
```diff
@@ -1,15 +1,11 @@
-import styles from "./Card.module.scss"
+import styles from "./Card.module.scss";
 
 interface Content {
-    children: any
+  children: JSX.Element | string | JSX.Element[];
```
Comment (FutureInventor) [inline src/components/Card/Card.tsx:4]: Great point :+1:

File: src/components/Card/Card.tsx [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
 }
 
 const Card = (props: Content) => {
```
Comment (FutureInventor) [inline src/components/Card/Card.tsx:7]:
```suggestion
const Card = (props: Content) => (
    <div className={styles.card}>
        {props.children}
    </div>
)
```

We do not have any logic in this component therefore we can use this shortcut. Add this in other suitable components as well.

File: src/components/Container/Container.test.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { render } from "@testing-library/react";
+import { it, describe, expect } from "vitest";
+import "@testing-library/jest-dom";
+import Container from "./Container";
+
+describe("Container", () => {
```
Comment (FutureInventor) [inline src/components/Container/Container.test.tsx:6]: Add test for the case when page is passed.

File: src/components/Container/Container.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import styles from "./Container.module.scss";
+
+interface Content {
```
Comment (FutureInventor) [inline src/components/Container/Container.tsx:3]:
```suggestion
interface IProps {
```

When we have interface that is specify for the props and it is not exported outside of the file, we can name it Prop. Add 'I' before the name is a good practice for naming interfaces overall.

Comment (FutureInventor) [reply src/components/Container/Container.tsx:3]: Add this chnge to other files as well.

File: src/components/Container/Container.test.tsx (hunk 4 above)
Comment (FutureInventor) [inline src/components/Container/Container.test.tsx:6]: Add test for the case when page prop is passed.

File: src/components/Header/Header.test.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { render } from "@testing-library/react";
+import { it, describe, expect } from "vitest";
+import "@testing-library/jest-dom";
+import Header from "./Header";
+
+describe("Header", () => {
```
Comment (FutureInventor) [inline src/components/Header/Header.test.tsx:6]: Add test for the case with page.

File: src/components/Item/Item.test.tsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { render } from "@testing-library/react";
+import { it, describe, expect } from "vitest";
+import "@testing-library/jest-dom";
+import Item from "./Item";
+
+describe("Item", () => {
```
Comment (FutureInventor) [inline src/components/Item/Item.test.tsx:6]: Add test for the page with page prop.

File: src/components/LinkToPage/LinkToPage.tsx [hunk 8]
```diff
@@ -8,8 +8,6 @@ interface LinkTo {
 }
 
 const LinkToPage = (props: LinkTo) => {
```
Comment (FutureInventor) [inline src/components/LinkToPage/LinkToPage.tsx:10]: Is 'urlTo' a required prop for sure? If yes the ternaty operator in 15 line will alwas cast to true and styles.active will be added.

File: src/components/LinkToPage/LinkToPage.test.tsx [hunk 9]
```diff
@@ -1,22 +1,44 @@
...
-  it("text 'Link' should be rendered", () => {
+vi.mock("react-router-dom", () => ({
+  Link: vi
+    .fn()
+    .mockImplementation((props: ComponentProps<typeof Link>) => (
+      <div data-testid={props.to}>{props.children}</div>
+    )),
+}));
+
+describe("LinkToPage", () => {
```
Comment (FutureInventor) [inline src/components/LinkToPage/LinkToPage.test.tsx:16]: Add to tests whenever urlTo is passed properly.

File: src/pages/Work/Work.tsx [hunk 10]
```diff
@@ -32,25 +35,23 @@ const Work = () => {
...
               </Box>
-            </div>
-            <div className={styles.item2}>
+            </Item>
+            <Item page="item2">
               <Box>
                 Currently, I am mainly focused on creating a web application for
                 a building depot. The project requires creating a database with
-                the current product list, which will be updated every time
+                the current product list, which will be upWorkdated every time
```
Comment (FutureInventor) [inline src/pages/Work/Work.tsx:43]:
```suggestion
                the current product list, which will be updated every time
```

**Comment by JJonka** [comment]
I have added changes after review with the latest commit

**Pull request merged**